Makefile

```
@@ -28,6 +28,9 @@ github-runner:
 build-image:
 	cd src && bash -x scripts/docker_image_build.sh
 
+build-spawner-image:
```

See comment on `src/scripts/docker_image_spawner_build.sh` file.

```suggestion
build-spawner-image: build
```